gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactorJobRunner.java

```diff
@@ -168,27 +167,12 @@ private void submit(Job job) throws ClassNotFoundException, IOException, Interru
 
   private void moveTmpPathToOutputPath() throws IOException {
     LOG.info(String.format("Moving %s to %s", this.tmpPath, this.outputPath));
+    this.fs.delete(this.outputPath, true);
```

Comment:
is there some protection against the user giving wrong parameters and accidentally deleting something they shouldn't?
I suggest having the old logic for most cases, and just having the different logic if inputPath == outputPath.